app/uk/gov/hmrc/emcstfe/models/response/prevalidate/ExciseTraderValidationResponse.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emcstfe.models.response.prevalidate
+
+import play.api.libs.json.{Format, Json}
+
+
+case class ProductErrors(
+                          exciseProductCode: String,
+                          errorCode: String,
+                          errorText: String
+                        )
+
+case class ValidateProductAuthorisationResponse(
+                                                 valid: Boolean,
+                                                 productError: Option[Seq[ProductErrors]] = None
+                                               )
+
+case class ExciseTraderResponse(
+                                 exciseRegistrationNumber: String,
```

Comment:
Spec says this is exciseRegsitrationNumber, but not sure on the real API behaviour

Reply:
Asked in the chat with EIS

Reply:
This was corrected in the API spec

Section 12.2.2.3, 12.2.2.4 - Corrected the field name exciseRegistrationNumber in Schema, Schema Diagram and Samples for Error Response

Reply:
The API spec is currently at version 1.5 (8th Jan 2024)